src/ICeWall/src/ICeWall.tcl

```diff
@@ -3021,9 +3189,9 @@ namespace eval ICeWall {
 
     for {set row 1} {$row <= $num_bumps_y} {incr row} {
       for {set col 1} {$col <= $num_bumps_x} {incr col} {
+        if {![bump_exists $row $col]} {continue}
         set origin [get_bump_origin $row $col]
         set bump_name [get_bump_name_at_row_col $row $col]
-        # debug "bump ($row $col): $bump_name"
 
         set inst [odb::dbInst_create $block $bump_master $bump_name]
```

Comment:
I think the bumps in the middle need to have pin names (ports) added to them. Ie. "VDD", "VSS", etc. so it's clear when looking at the DEF what bump goes with what signal.